app/representers/cart_representer.rb

```diff
@@ -1,18 +1,18 @@
 module CartRepresenter
   include Roar::JSON::HAL
 
-  property :type, getter: ->(a) {'cart'}, writeable: false
+  property :type, getter: ->(_) {'cart'}, writeable: false
   property :id, writeable: false
   property :user_id
 
-  collection :cart_items, class: CartItem do
-    property :type, getter: ->(a) {'cart_item'}, writeable: false
-    property :pricing_id
-    property :quantity
-    property :product_name, writeable: false
-    property :unit_price, writeable: false
-    property :total_price, writeable: false
-  end
+  collection :cart_items, extend: CartItemRepresenter, class: CartItem,
+    parse_strategy: ->(fragment, _, _) {
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Avoid using {...} for multi-line blocks.